x/erc20/types/codec.go

```diff
@@ -63,10 +56,10 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 // concrete types on the provided LegacyAmino codec. These types are used for
 // Amino JSON serialization and EIP-712 compatibility.
 func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
-	cdc.RegisterConcrete(&MsgRegisterCoin{}, "canto/RegisterCoinProposal", nil)
-	cdc.RegisterConcrete(&MsgRegisterERC20{}, "canto/RegisterERC20Proposal", nil)
-	cdc.RegisterConcrete(&MsgConvertERC20{}, convertERC20Name, nil)
-	cdc.RegisterConcrete(&MsgConvertCoin{}, convertCoinName, nil)
-	cdc.RegisterConcrete(&MsgUpdateParams{}, msgUpdateParams, nil)
-	cdc.RegisterConcrete(&Params{}, "canto/Params", nil)
+	cdc.RegisterConcrete(&MsgRegisterCoin{}, "canto/MsgRegisterCoin", nil)
+	cdc.RegisterConcrete(&MsgRegisterERC20{}, "canto/MsgRegisterERC20", nil)
+	cdc.RegisterConcrete(&MsgConvertCoin{}, "canto/MsgConvertCoin", nil)
+	cdc.RegisterConcrete(&MsgConvertERC20{}, "canto/MsgConvertERC20", nil)
```

Comment:
@dudong2 
Those lines are redundant. Any reason for this?

Another question:
- Why don't we use `canto/x/erc20/MsgRegisterCoin` and `canto/x/erc20/MsgRegisterERC20` here?
- I know it follows `amino.name` in `tx.proto`, but curious for naming. I think you were trying to unify the naming.

Reply:
good opinion. but we need to consider consistency not only with canto but also with cosmos-sdk.
in the sdk, it follows the form `cosmos-sdk/Msg...`, so Canto should also adopt this format.

Reply:
@dudong2 thanks for your details :)

now LGTM